template-model-generator/src/main/kotlin/no/nav/pensjon/brev/template/dsl/helpers/TemplateModelVisitor.kt

```diff
@@ -33,10 +35,17 @@ internal class TemplateModelVisitor(
         throw MissingImplementation("Couldn't process node $node at: ${node.location}", node)
     }
 
+    private fun KSClassDeclaration.notInPackage(name: String): Boolean {
+        val pkg = this.packageName.asString()
+        val nameWithSeparator = if (name.endsWith(".")) name else "$name."
+
+        return pkg != name && !pkg.startsWith(nameWithSeparator)
+    }
+
     override fun visitClassDeclaration(classDeclaration: KSClassDeclaration, data: SelectorModels): SelectorModels =
         if (data.isVisited(classDeclaration)) {
             data.withDependency(classDeclaration, dependency)
-        } else if (classDeclaration.classKind == ClassKind.CLASS && classDeclaration.modifiers.contains(Modifier.DATA)) {
+        } else if (classDeclaration.classKind in setOf(CLASS, INTERFACE) && classDeclaration.notInPackage("java") && classDeclaration.notInPackage("kotlin")) {
```

Comment:
No vil vi vel også generere opp selectors for klassar som ikkje er data-klassar, men det er vel kanskje heilt greitt?

Reply:
Ja, det var det jeg tenkte. Det er jo en litt merkelig begrensning i utgangspunktet.